### DISCUSSION TITLE
Welcome to riverui Discussions!

### DISCUSSION THREAD

**Discussion by bgentry** [riverqueue/riverui#49] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions about how to use River UI, or if you're running into something unexpected.
  * Share ideas and feature requests.
  * Engage with other community members.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hyrefeldt-ourstudio** [comment]
Didn't see this Welcome post until now. Maybe time to introduce myself a bit then.

I'm living and working in Gothenburg, Sweden. Right now, as a Go developer contracted to Volvo Cars.

I discovered Riverqueue via a colleague that told me about it and that it could fit our use cases to handle long running jobs. So far, it is quite nice to use and has solved some problems for us already. Looks like you've done a good job building this tool!

Best regards,
Johan Hyrefeldt

**Reply by bgentry** [reply]
@hyrefeldt-ourstudio that’s such a cool use case! Thanks for sharing and great to meet you. 

Please feel free to reach out as you encounter issues, or if you think of missing features that would save you a lot of time / make life easier. 🙏🏻

**Reply by hyrefeldt-ourstudio** [reply]
Yeah! Great to meet you too!

I will do! We see a couple of more usecases for Riverqueue in our system which provides capabilities to configure cars on volvocars.com. We haven't had time to implement it in all places yet though.

**Comment by nexovec** [comment]
Also didn't see this before for some reason.

I'm a one-man team from Czech Republic working on a small educational software product on the side of work and studies. It is called chatlearning and it is a modernized version of [anki](https://apps.ankiweb.net/). It offers tests with open ended answers(via llms), social and realtime features, and a personally tailored feedback and repetition algorithms.

I'm using river since the start of 2024, I think it is a godsend for simplifying my operations(app itself only has postgres as a dependency) without sacrificing scalability, features or performance. Massively useful for real-time background processing.

This is some cool tech, thank you for your effort.

**Reply by brandur** [reply]
Very cool @nexovec. Thanks for checking in!
